### DISCUSSION TITLE
SpoofDPI запускается, но сайты по-прежнему не открываются.

### DISCUSSION THREAD

**Discussion by DDDstart** [xvzc/SpoofDPI#107] [Q&A] [answered]
SpoofDPI устанавливается нормально. Запускается. Когда терминал работает с запущенным SpoofDPI, я запускаю Firefox с настройками прокси, как указано на скриншоте. Но сайты по-прежнему не открываются.
Что не так?

![Screenshot](https://github.com/user-attachments/assets/19d1cc4b-f896-4acd-8b13-f758ebed1be3)
![Screenshot2](https://github.com/user-attachments/assets/81077926-990d-437c-8aec-ac9fc022de3d)

**Comment by lovipomidorku** [comment]
Насколько я знаю, это работает только с браузером Chrome. 

Также, слышал что в Chrome нужно отключить функцию `TLS 1.3 hybridized Kyber support` в настройках по этому пути: `chrome://flags`. 

Но у меня почему-то работает без выключения этой настройки, а также в Safari. Firefox не проверял

**Reply by DDDstart** [reply]
Поставил Chrome, отключил TLS 1.3.
Запускаю так, как в инструкции: google-chrome --proxy-server="http://127.0.0.1:8080". Chrome запускается, но сайты все равно не открываются. Открываются только те, что открывались и ранее, как-будто бы ничего т не поменялось.

**Reply by flytaly** [reply, marked as answer]
У меня тоже не открываются.  Конкретно это средство помогает только против замедления ютуб. А так и я использовал "zapret"  на линуксе вот он позволяет и замедление ютуб обойти и блокировку сайтов. 
В моем случае с "nfqws --dpi-desync=fake,split2 --dpi-desync-ttl=5". 

https://www.youtube.com/watch?v=FdDC9R7gL-Y

**Comment by PoorMouse** [comment]
Никакую настройку прокси в браузере делать не нужно.

**Comment by EndlessNight191** [comment]
После установки spoofDpi на macos перестал открываться postman, slack. Сафари не открывает ни один сайт.

Выключил spoofDpi через kill процесса. Но это не помогло. Кто-то сталкивался с таким? Как решали?

**Reply by Aktaviys** [reply]
У меня была такая проблема но я удалял все настройки вайфай https://youtu.be/2X6XVcATO-8?si=Z9jKdTJEFjYNdmol

**Comment by zeetech1012** [comment]
ubuntu 24 ютюб открывает и с включенным и выключенным а вот видое не грузит ни так ни так

**Comment by gray1717** [comment]
хоть бы написали какие именно сайты не открываются. У меня firefox на linux, бывает что-то со второго, третьего раза только открывается.
НО, я себе создал pac файл и в нем прописал какие сайты открывать через spoof-dpi, все остальное напрямую

**Reply by EndlessNight191** [reply]
Не открываются абсолютно все сайты именно через сафари, edge открывает половину сайтов (разные корпоративные доски и т.д тоже не открывал, даже через edge, а вот в duckduck все открывалось но с 3 раза и в инкогнито)

Можете поделится pac файлом, как вы его настраивали?

**Reply by gray1717** [reply]
> Не открываются абсолютно все сайты именно через сафари, edge открывает половину сайтов (разные корпоративные доски и т.д тоже не открывал, даже через edge, а вот в duckduck все открывалось но с 3 раза и в инкогнито)
> 
> Можете поделится pac файлом, как вы его настраивали?

````
function FindProxyForURL(url, host){
    if (dnsDomainIs(host, ".youtube.com") || dnsDomainIs(host, ".googlevideo.com") || dnsDomainIs(host, ".ytimg.com") || dnsDomainIs(host, "rutracker.org") || dnsDomainIs(host, "rutracker.cc")) {
        return "PROXY 192.168.1.1:8080"; }
    return "DIRECT";
}

````

создаете файл с расширением pac, и туда этот текст, в настройках браузера прописываете путь до этого файла, у меня firefox, как на скриншоте в первом посте
upd.: URL автоматической настройки proxy
